customersupport/models.py

```diff
@@ -252,3 +248,101 @@ def replace_date(self):
+    Tickets expose the following data
+    Id of the ticket:           id
+    Issue Enumeration:          issue
+    Date Opened:                date_opened
+    Date Closed:                date_closed
+    Current Status Enumeration: current_status
+    List of Call Sessions:      sessions
+    Customer on Ticket:         customer_id
+    Order on Ticket:            order_id
+
+    """
+    __tablename__ = 'ticket'
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    issue = Column(Enum(TicketType))
+    date_opened = Column(DateTime)
+    date_closed = Column(DateTime)
+    current_status = Column(Enum(TicketStatus))
+    sessions = relationship('CallLog',backref="post",cascade="all, delete-orphan",lazy="dynamic")
+    customer_id = Column(String(16)) 
+#    order_id = Column(String(60),ForeignKey('order.id'))
+    
+    def __init__(self, ticket_dict):
+        """
+        Params come from API docs
+        :param ticket_dict: dictionary for init ticket
+        """
+        self.issue = ticket_dict["issue"]
+        self.date_opened = ticket_dict["date_opened"]
+        self.date_closed = ticket_dict["date_closed"]
+        self.current_status = ticket_dict["current_status"]
+        self.sessions = ticket_dict["sessions"]
+        self.customer_id = ticket_dict["customer_id"]
+#        self._order_id = ticket_dict["order_id"]
+
+    def __repr__(self):
+        return '<Ticket %r %r %r %r %r %r>' %(self.id, self.issue, self.date_opened, self.current_status, self.sessions, self.customer_id)
+
+    def serialize(self):
+        sessions = []
+        for session in self.sessions:
+            print(session)
+            sessions.append(session.serialize())
+        return {
+            "id": self.id,
+            "issue": self.issue.name,
+            "date_opened": self.date_opened.strftime('{%Y-%m-%d %H:%M:%S}'),
+            "date_closed": self.date_closed.strftime('{%Y-%m-%d %H:%M:%S}'),
+            "current_status": self.current_status.name,
+            "sessions": sessions,
+            "customer_id": self.customer_id
+        #    "order_id": self.order_id
+        }
+
+class CallLog(Base):
+    __tablename__ = "call_log"
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    date_called = Column(DateTime)
+    calling_number = Column(String(60))
+    callback_number = Column(String(60))
+    notes = Column(String(500))
+#    action_taken = Column(ActionTaken)????
+    employee = Column(String(16))
+    ticket = Column(Integer, ForeignKey('ticket.id'))
+
+    def __init__(self, call_dict):
+        self.date_called = call_dict["date_called"]
+        self.calling_number = call_dict["calling_number"]
+        self.callback_number = call_dict["callback_number"]
+        self.notes = call_dict["notes"]
+        self.employee = call_dict["employee"]
+
+    def __refr__(self):
+        return '<CallLog %r $r $r $r %r %r>' %(self.date_called,self.calling_number,self.callback_number,self.notes,self.employee)
```

Comment:
I think there are a few typos on this line (unless I'm missing something).
- `__refr__` should be `__repr__`
- `$r` should be `%r` (or a `.format` string, like I mentioned in an earlier comment.)